### DISCUSSION TITLE
API Link to get current top emotes?

### DISCUSSION THREAD

**Discussion by toasterudel** [night/betterttv#5867] [Q&A] [answered]
Currently there is an end point on the API to get all emote data on the "Global Emotes" tab with https://api.betterttv.net/3/cached/emotes/global. Is there a similar end point to get all the emotes on the "Top Emotes" tab?

**Comment by night** [comment, marked as answer]
We don't have a documented API at this time for this information. We do have an experimental OAuth2 API which can give access to this. If you have a unique use case for API access please reach out to us at https://nightdev.com/contact?product=betterttv&topic=business_development
